packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -592,6 +594,11 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
     return renderedItems;
   };
 
+  private _defaultMenuItemRenderer = (item: IContextualMenuItemRenderProps): React.ReactNode => {
+    const { index, focusableElementIndex, totalItemCount, hasCheckmarks, hasIcons, ...rest } = item;
+    return this._renderMenuItem(rest, index, focusableElementIndex, totalItemCount, hasCheckmarks, hasIcons);
```

Comment:
actually, i think if we do the following we can get good perf for default renderer case too:
```
    const { index, focusableElementIndex, totalItemCount, hasCheckmarks, hasIcons } = item;
    return this._renderMenuItem(item, index, focusableElementIndex, totalItemCount, hasCheckmarks, hasIcons);
```